aerofiles/seeyou/reader.py

```diff
@@ -68,6 +81,23 @@ def decode_waypoint(self, fields):
             'description': self.decode_description(fields[10]),
         }
 
+    def decode_task_info(self, tasks, fields):
+        if fields[0] == 'Options':
+            tasks[-1]['Options'] = self.decode_task_options(fields)
+        elif fields[0].startswith('ObsZone'):
+            tasks[-1]['ObsZones'].append(self.decode_task_ObsZone(fields))
+        else:
```

Comment:
if think I'd prefer it if this if/else block was in `read()`, so that `decode_task_info` only decodes the single task line and doesn't mutate `tasks` directly